src/modules/BuilderPlace/hooks/platform/useGetPlatform.tsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react';
+import { getPlatformBy } from '../../request';
+import { BuilderPlace } from '@prisma/client';
+
+const useGetPlatformByOwnerId = (
```

Comment:
be careful, this is V1 like hook without the best practices: 
- use react query to prevent reinventing the wheel on loading, error management etc (cf useServiceById)
- creating a hook for one precise use case, here a filter by OwerId. In this PR there are 7 filters, it would mean 7 hooks with 99% of code share. So just create a hook useGetPlatformBy taking an array of filters corresping to the new typing PlatformsFilters